addons/website/static/src/scss/website.scss

```diff
@@ -1433,6 +1433,14 @@ $ribbon-padding: 100px;
     :not(.o_cookies_popup) {
         bottom: 0;
     }
+    .modal-dialog {
+        margin: 0
```

Comment:
if I understand the issue that the modal take more height than page happen because we have:

```css
.s_popup[data-vcss='001'] .modal-dialog {
    height: auto;
    min-height: 100%;
}
.modal-dialog {
    position: relative;
    width: auto;
    margin: 0.5rem;
    pointer-events: none;
}
```
if seems to me that it should be solved for all `.s_popup[data-vcss='001'] .modal-dialog` not just this one.

there is also for .s_popup this CSS that is removing the right margin but not the left one that is a little odd:

```css
.s_popup[data-vcss='001'] .s_popup_top .modal-dialog, .s_popup[data-vcss='001'] .s_popup_bottom .modal-dialog {
    margin-right: 0;
}
```

Reply:
I completely agree on this point. I'm going to find out why it wasn't done before and/or it was intentional before making this PR ready.
Thanks!

Reply:
I redid some tests and it turns out that we have to remove all the margins here if you want to reproduce the behavior in desktop view and be able to navigate through the site despite the cookiebar. Otherwise the scroll is done only on the cookiebar and we are unable to click away.
Also, making the change here to `.s_popup` (bottom, top) also improves the display of popups in mobile view (at least according to the few tests I did).

So I updated the PR according to this the your last comments